cfme/infrastructure/config_management/satellite.py

```diff
@@ -105,6 +106,11 @@ def from_config(cls, prov_config, prov_key, appliance=None):
         )
 
 
+@attr.s
+class SatelliteProvidersCollection(ConfigManagerProviderCollection):
+    ENTITY = SatelliteProvider
```

Comment:
In the same way there is no e.g. `VMWareProvidersCollection` or `AzureProvidersCollection` I think that there should be no `SatelliteProvidersCollection`. I understand that you've added this in for the `test_advanced_search` but is there anyway we can workaround the issue without adding a `SatelliteProvidersCollection`? For instance could you pass the provider type as an optional param in the `SearchParams` named tuple?

Reply:
@psimovec should we remove this now?

Reply:
yes we should, thanks
I have deleted it from entry points but forgot to delete it from this location